runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -108,7 +109,7 @@ public IStreamView copy(UUID source, UUID destination, long timestamp) {
      * @throws TransactionAbortedException If the transaction was aborted by
      *                                     the sequencer.
      */
-    public long append(@Nonnull Set<UUID> streamIDs, @Nonnull Object object,
+    public long append(@Nonnull List<UUID> streamIDs, @Nonnull Object object,
                        @Nullable TxResolutionInfo conflictInfo) throws TransactionAbortedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.TransactionAbortedException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)